release-notes/supported-os.json

```diff
@@ -0,0 +1,107 @@
+{
+    "$schema": "https://json.schemastore.org/dotnet-supported-os.json",
+    "operating-systems": [
+        {
+            "name": "Ubuntu",
+            "lifecycle-policy": "https://wiki.ubuntu.com/Releases",
+            "last-updated": "2023-02-28",
+            "upcoming-eos": {
+                "version": "23.10",
+                "eos-date": "2024-07-01"
+            },
+            "versions": [
```

Comment:
That works for me.

Reply:
I think keeping them is important as the `unsupported-versions` adds value. The difference between `only unsupported-versions` and the cycle being removed would be, that you can see which versions had been supported in the past. This would be lost on removal.

Use-case: An existing customer still uses 18.04 and asks what's the latest version he could use.

About EOL/EOS: I prefer using the term End of Support (EOS) over End of Life (EOL) specifically for that reason. EOL usually includes ESM/ESU/ELTS/... (however each distribution calls their extended support lifecycles) while EOS usually is only standard support.

We COULD add the end of life per cycle which would avoif confusion on which EOS we use. It would just add a bit more maintenance effort as some of them are not cleanly communicated (i.e. "end of May 2024"). I usually tend to use last day of the month for those cases. If it's only "May 2024" I use the first day.